.codespellrc

```diff
@@ -1,5 +1,5 @@
 [codespell]
-skip = .git,node_modules,venvs,.tox,yarn.lock,CHANGELOG.md
+skip = .git,node_modules,venv,.venv,.mypy_cache,.tox,yarn.lock,CHANGELOG.md,./htmlcov,*.pyc
```

Comment:
Ran locally an found it complaining about some files/folders in the web directory

```suggestion
skip = .git,node_modules,dist,venv,.venv,.mypy_cache,.tox,yarn.lock,CHANGELOG.md,./htmlcov,*.pyc,*.log
```